### ISSUE TITLE
Extension fails accessibility

### ISSUE THREAD

**Issue by pabrams** [microsoft/vsts-extension-retrospectives#368]
#41 may be resolved by this.

The accessibility team at ESDC performed an assessment of this extension, and it failed. What are the chances of having these addressed by Microsoft?

<html>
<body>
<div><span style="background-color: rgb(249, 249, 249); color: rgb(51, 51, 51); font-family: Helvetica, Arial, sans-serif; font-size: 34px; font-style: normal; font-weight: 700; hyphens: manual; orphans: 2; text-align: start; text-decoration-line: none; vertical-align: baseline; white-space: normal; widows: 2;">Assessment for Retrospectives Azure DevOps Extension, Retrospectives Azure DevOps Extension</span></div><div><span style="background-color: rgb(249, 249, 249); color: rgb(51, 51, 51); font-family: Helvetica, Arial, sans-serif; font-size: 16px; font-style: normal; font-weight: 400; hyphens: manual; orphans: 2; text-align: start; text-decoration-line: none; vertical-align: baseline; white-space: normal; widows: 2;">Performed on 2022-05-13</span></div><div><span style="background-color: rgb(249, 249, 249); color: rgb(51, 51, 51); font-family: Helvetica, Arial, sans-serif; font-size: 18px; font-style: normal; font-weight: 700; hyphens: manual; orphans: 2; text-align: start; text-decoration-line: none; vertical-align: baseline; white-space: normal; widows: 2;">Current Scores for Audit</span></div><div><span style="background-color: rgb(249, 249, 249); color: rgb(51, 51, 51); font-family: Helvetica, Arial, sans-serif; font-size: 16px; font-style: normal; font-weight: 400; hyphens: manual; orphans: 2; text-align: start; text-decoration-line: none; vertical-align: baseline; white-space: normal; widows: 2;">Accessibility Score: </span><div><span style="background-color: rgb(249, 249, 249); color: rgb(51, 51, 51); font-family: &quot;Glyphicons Halflings&quot;; font-size: 16px; font-style: normal; font-weight: 400; hyphens: manual; orphans: 2; text-align: start; text-decoration-line: none; vertical-align: baseline; white-space: normal; widows: 2;"></span></div></div><div><span style="background-color: rgb(249, 249, 249); color: rgb(51, 51, 51); font-family: Helvetica, Arial, sans-serif; font-size: 16px; font-style: normal; font-weight: 700; hyphens: manual; orphans: 2; text-align: start; text-decoration-line: none; vertical-align: baseline; white-space: normal; widows: 2;">Reviewed By</span></div><dl><dt><div><span style="background-color: rgb(249, 249, 249); color: rgb(51, 51, 51); font-family: Helvetica, Arial, sans-serif; font-size: 16px; font-style: normal; font-weight: 700; hyphens: manual; orphans: 2; text-align: start; text-decoration-line: none; vertical-align: baseline; white-space: normal; widows: 2;">RMT ID</span></div></dt><dd><div><span style="background-color: rgb(249, 249, 249); color: rgb(51, 51, 51); font-family: Helvetica, Arial, sans-serif; font-size: 16px; font-style: normal; font-weight: 400; hyphens: manual; orphans: 2; text-align: start; text-decoration-line: none; vertical-align: baseline; white-space: normal; widows: 2;">a11y-5658</span></div></dd><dt><div><span style="background-color: rgb(249, 249, 249); color: rgb(40, 65, 98); font-family: Helvetica, Arial, sans-serif; font-size: 16px; font-style: normal; font-weight: 400; hyphens: manual; orphans: 2; text-align: start; text-decoration-line: underline; vertical-align: baseline; white-space: normal; widows: 2;">https://a11yrmt.ca/viewrequest-en.php?erid=NTY5NQ==</span></a></div></dd></dl>

  |   | Number of pages that fail
-- | -- | --
ARIA attributes must conform to valid values | https://dequeuniversity.com/rules/axe/3.5/aria-valid-attr-value, Blind, Deafblind, Mobility | 1
Scope attribute should be used correctly on tables | https://dequeuniversity.com/rules/axe/3.5/scope-attr-valid, Blind, Deafblind, Mobility | 1
Text elements must have sufficient color contrast against the background | https://dequeuniversity.com/rules/axe/3.5/color-contrast, Low Vision, Colorblindness | 1
Content shown on hover not being hoverable or being difficult to hover | https://www.w3.org/WAI/WCAG21/Techniques/failures/F95.html, Low-Vision, Cognitive | 1
Interactive component not keyboard operable | WCAG SC 2.1.1 - Keyboard; blind, Deafblind, Mobility | 1
Focus order unpredictable | Focus order unpredictable; Blind, Deafblind, Mobility | 1
Focus Indicator Disappears While Tabbing | Focus indicator disappears while tabbing through the page | 1
Page is not valid HTML | Page does not pass W3C validation in a way described by 4.1.1 - Parsing | 1
Bad or no reflow on some screen resolutions | Content can be presented without loss of information or functionality, and without requiring scrolling in two dimensions | 1
Empty <th> should be <td> | Blind,Deafblind | 1
Elements on page in illogical order | Blind, Deafblind, Mobility, Cognitive | 1

**Comment by polatengin** [comment]
resolved with many PRs, one of the last ones is; #1437
